src/mpl2/src/rtl_mp.cpp

```diff
@@ -94,94 +101,100 @@ bool rtl_macro_placer(const char* config_file,
   string block_file = string(report_directory) + '/' + report_file + ".block";
   string net_file = string(report_directory) + '/' + report_file + ".net";
 
-  // parameters defined in config_file
+  //
+  //  Default for Parameters
```

Comment:
Can these parameters be made const?

Reply:
As the these defaults are overwritten by the config file if used, they cannot be constants.